### ISSUE TITLE
Media: 3P assets do not include ALT attributes

### ISSUE THREAD

**Issue by spacedmonkey** [GoogleForCreators/web-stories-wp#7273]
## Bug Description

None of the 3P assets from tenor, unsplash or coverr include an alt attribute when rendered or inserted in a page. 

This data does not seem to be included in api either. 

Discovered as part of #7272

## Expected Behaviour

Third party access should have alt from the original source, ie unsplash. This is useful for SEO and screen readers. 

## Screenshots

![Screenshot 2021-04-21 at 17 09 31](https://user-images.githubusercontent.com/237508/115586402-7549e980-a2c4-11eb-8332-80afff2c6bfb.png)
![Screenshot 2021-04-21 at 17 09 15](https://user-images.githubusercontent.com/237508/115586408-7713ad00-a2c4-11eb-8909-ede1e6d87a89.png)
![Screenshot 2021-04-21 at 17 08 52](https://user-images.githubusercontent.com/237508/115586410-77ac4380-a2c4-11eb-9b1b-20398a89413c.png)

![Screenshot 2021-04-21 at 17 12 02](https://user-images.githubusercontent.com/237508/115586690-c3f78380-a2c4-11eb-880e-4723230f6725.png)

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
@choumx we'd need to add this to the media3p API (assuming that we get this data from the providers). If they don't provide any human readable description that we could use for `alt` text, we need an alternative solution. Any suggestions?

**Comment by dreamofabear** [comment]
Good catch, looks like Unsplash and Tenor both should support `alt`. We'd need to plumb this through media3p as you suggest.

**Comment by spacedmonkey** [comment]
Coverr has title and description fields. These could map to a alt. 

https://api.coverr.co/docs/videos/#list-videos

Unsplash has a field called `alt_description` in the api.

**Comment by swissspidy** [comment]
Created [b/187693612](b/187693612) for this

**Comment by swissspidy** [comment]
This is now unblocked. Title & Description should be readily available via the media3p API

**Comment by spacedmonkey** [comment]
Which field should we use here?

**Comment by swissspidy** [comment]
`description` with a fallback to `title` if missing
